app/assets/javascripts/map/gmap.js

```diff
@@ -1,13 +1,15 @@
-var geocoder;
-// var address = "San Diego, CA";
-
-function initMap() {
-  var map = new google.maps.Map(document.getElementById("map-container"), {
-    center: {lat: 43.825592, lng: 87.616848},
-    zoom: 2,
-    mapTypeId: google.maps.MapTypeId.HYBRID
+// var geocoder;
+function gmap_show(rawMarkers) {
```

Comment:
Identifier 'gmap_show' is not in camel case.<br>'gmap_show' is defined but never used.